### ISSUE TITLE
Course 3 - Week 2 - Lesson 3.ipynb

### ISSUE THREAD

**Issue by rbhambriiit** [lmoroney/dlaicourse#134]
https://github.com/lmoroney/dlaicourse/blob/master/TensorFlow%20In%20Practice/Course%203%20-%20NLP/Course%203%20-%20Week%202%20-%20Lesson%203.ipynb

history = model.fit(train_data, epochs=num_epochs, validation_data=test_data)
hits error.

fix:
replace train_data with train_dataset
replace test_data with test_dataset

**Comment by rbhambriiit** [comment]
also i see:

Epoch 10/10
391/391 [==============================] - 16s 41ms/step - loss: 0.1622 - accuracy: 0.9431 - val_loss: 0.3225 - val_accuracy: 0.8786


course video expects this to be ~60%

**Comment by juan-coursera** [comment]
Hi rbhambriiit,

Yes, it should be train_dataset and test_dataset. This was a typo on my part. I made a pull request [#110](https://github.com/lmoroney/dlaicourse/pull/110) back in June 12th to fix this. I'm waiting for it to be merged.
